tests/Tokenizer/CodeHasherTest.php

```diff
@@ -26,7 +26,10 @@ final class CodeHasherTest extends TestCase
 {
     public function testCodeHasher(): void
     {
-        self::assertSame('d9de49676ba2316990a5acd04c8418e8', CodeHasher::calculateCodeHash('<?php echo 1;'));
-        self::assertSame('d9de49676ba2316990a5acd04c8418e8', CodeHasher::calculateCodeHash('<?php echo 1;')); // calling twice, hashes should always be the same when the input doesn't change.
+        $code = '<?php echo 1;';
+        $hash = hash('xxh128', $code);
+
+        self::assertSame($hash, CodeHasher::calculateCodeHash($code));
+        self::assertSame($hash, CodeHasher::calculateCodeHash($code)); // calling twice, hashes should always be the same when the input doesn't change.
```

Comment:
your `expected` and `actual` are same logic copy-pasted.
let's have `expected` as provided constant value, as originally.